pom.xml

```diff
@@ -599,7 +599,7 @@
       <dependency>
         <groupId>org.glassfish.jaxb</groupId>
         <artifactId>jaxb-runtime</artifactId>
-        <version>2.3.2</version>
+        <version>4.0.5</version>
```

Comment:
is there version contract between this dep and other `org.glassfish.*` deps?

Reply:
There is currently no mutual dependence between the `org.glassfish.jersey` series (3.0.16) and the `org.glassfish.jaxb` (4.0.5) series.

But they both depend on `jakarta.xml.bind:jakarta.xml.bind-api` and `jakarta.activation:jakarta.activation-api`, and the dependency versions of `jaxb` are newer (xml: 4.0.2 vs 3.0.1, activation: 2.1.3 vs 2.0.1).

Reply:
I mean `com.sun.xml.fastinfoset:FastInfoset` and `org.jvnet.staxex:stax-ex`, they are marked as optional in `org.glassfish.jaxb:jaxb-runtime:4.0.5`